### DISCUSSION TITLE
Slider for animation

### DISCUSSION THREAD

**Discussion by phovorka** [google/model-viewer#3479] [Q&A]
Anyone can share solution for slider for animation? Sample or anything. Would be helpfull for me... .
Thanks .

**Comment by phovorka** [comment]
i forgott to say PLEASE :-)

**Reply by vaggarath** [reply]
What do you mean by slider ? Something to cycle through models ?

**Comment by phovorka** [comment]
something like this in editor-> 
![image](https://user-images.githubusercontent.com/67637867/169476008-bda010bb-b5b4-4b56-bd9d-c75053c82f5c.png)
with input type -range button

**Reply by elalish** [reply]
Our editor is open source too, so you could try here: https://github.com/google/model-viewer/blob/master/packages/space-opera/src/components/animation_controls/animation_controls.ts

**Reply by Goodsell3492** [reply]
I'm new to this as well not sure how to insert a slider with the link example.

**Reply by elalish** [reply]
Once you find or make an HTML slider UI, just link it's value changed event to set `mv.currentTime`. You'll also want `mv.duration` to know the end time.
